ui/components/multichain/network-list-menu/network-list-menu.tsx

```diff
@@ -506,12 +513,15 @@ export const NetworkListMenu = ({ onClose }: { onClose: () => void }) => {
     const { onDelete, onEdit, onDiscoverClick, onRpcConfigEdit } =
       getItemCallbacks(network);
     const iconSrc = getNetworkIcon(network);
+    const name = network.isEvm
+      ? network.name
+      : MULTICHAIN_NETWORK_TO_NICKNAME[network.chainId];
```

Comment:
This can be reverted since the word `mainnet` was removed

Reply:
https://github.com/MetaMask/metamask-extension/pull/31702/commits/ccb2b0591e5788137adc195aa66e6a6269a53bc7